protoc-gen-kotlin/lib/build.gradle.kts

```diff
@@ -31,6 +31,12 @@ kotlin {
             }
         }
 
+        val jvmMain by getting {
```

Comment:
The `BootstrapConverter` in protoc-gen-kotlin uses protobuf-java to bootstrap code generation if pbandk-generated code is not available. Honestly that code probably is broken since I've never once run it and it hasn't been tested in ages. But I didn't want to deal with pulling out that code in this PR, so I decided it was easier to just keep the `protobuf-java` dependency for the code generator for now. I don't think it should cause any problems since it's not used in the pbandk runtime library.